lua/starfall/transfer.lua

```diff
@@ -143,12 +143,76 @@ else
 		net.SendToServer()
 	end
 
-	net.Receive("starfall_upload", function(len)
+	-- BUG: Current implementation does not care about comments/strings, therefore it will end up searching/replacing --@using everywhere.
```

Comment:
This still needs to be addressed, but I am not entirely sure how.
I want to do this through `SF.Preprocessor.SetGlobalDirective` (like all other directives are being made), but it seems that a callback you pass to `SF.Preprocessor.SetGlobalDirective` is actually called by SF editor too, which is unwanted (`fileName` is equal to `"file"` when this happens...)
I think we need to introduce `SF.Postprocessor` for this to work, not a `Preprocessor`... 😆